### ISSUE TITLE
Incorrect remapping for sourcemaps with relative source paths

### ISSUE THREAD

**Issue by zxbodya** [SitePen/remap-istanbul#70]
Hi, I have caught an issue while remapping coverage report with source maps with relative source paths inside.

Issue appears when there are files nested at more than one level, deep:

```
lib/
├── Router.js
├── Router.js.map
├── ....
├── expressions
│   ├── compile.js           <----- files like this one would be remapped with incorrect paths
│   ├── compile.js.map
│   ├── ....
src/
├── Router.js
├── ....
├── expressions
│   ├── compile.js
│   ├── ....
```

file `lib/expressions/compile.js` happens to be remaped to path `../../src/expressions/compile.js` instead of `../src/expressions/compile.js`

This results in incorrect coverage info posted on codecov.io and broken html report like in issue #49

Project where you can reproduce this can be found here: https://github.com/zxbodya/router1

**Comment by zxbodya** [comment]
I have made workaround locally - replacing relative paths in source map with appropriate absolute paths after source map file is loaded. And it seems to work pretty well.

Should I create pull request with it?

**Comment by dylans** [comment]
Closed via 8d6b5a1
